src/components/Blog/constants/categories.tsx

```diff
@@ -7,6 +9,8 @@ export interface CategoryInterface {
     hideFromNavigation?: boolean
 }
 
+export const homeCategories = ['Guides', 'CEO diaries', 'Engineering', 'HogMail', 'Privacy', 'Product updates']
```

Comment:
@andyvan-ph The categories that appear on the blog home page can be changed here! They appear in order.

Reply:
Random question... in principle, would it possible for a blog post to have more than one category?